index.html

```diff
@@ -5663,14 +5663,27 @@ <h3>Algorithm</h3>
         <ol>
           <li>Create a new {{Promise}} <var>promise</var> and return it.
             The following steps are then deferred.</li>
+          <li class="changed">If the provided <a data-lt="jsonldprocessor-compact-input">input</a>
+            is a <a>RemoteDocument</a>,
+            initialize it as <var>remote document</var>.</li>
```

Comment:
Although I see what you mean, I find this phrasing odd. You initialize a *variable*, not the object that you set to that variable...
```suggestion
            initialize <var>remote document</var> to <a data-lt="jsonldprocessor-compact-input">input</a>.</li>
```